app/assets/stylesheets/modifysubject.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the modifysubject controller here.
```

Comment:
remove unneccessary files if you are not adding any css same for all the js files